src/gui/transferlistfilterswidget.cpp

```diff
@@ -574,16 +574,25 @@ TransferListFiltersWidget::TransferListFiltersWidget(QWidget *parent, TransferLi
     scroll->setWidget(frame);
     vLayout->addWidget(scroll);
     setLayout(vLayout);
-
-    QCheckBox *statusLabel = new QCheckBox(tr("Status"), this);
+		
+		auto IconCheckBox = [&](auto &&name) {
+			QCheckBox *c = new QCheckBox(name, this);
+			c->setStyleSheet("QCheckBox::indicator { width: 16px; height: 16px; }"
+											 "QCheckBox::indicator:checked { image: url(:icons/qbt-theme/drop-down.svg); }"
```

Comment:
You should try use the expand/collapse icon from QTreeWidget/QTreeView for native widget feeling.

Reply:
what do you suggest, free function or calling `setStyleSheet` for each checkbox

Reply:
I tried but `expand.svg` is not loading for some reason, idk

Reply:
my bad, expand.svg was not in icons.qrc
@all which one you prefer?
this one
![image](https://user-images.githubusercontent.com/34717789/57969096-7f546680-7990-11e9-976f-d328690527a3.png)
or one in the first post?

Reply:
neither, I meant you should load the native one from OS
win7 example: https://i.stack.imgur.com/vE7fi.png
win10 example: https://cloud.addictivetips.com/wp-content/uploads/2015/07/win10-autoplay.png

Reply:
i know about `QIcon::fromTheme`, but i don't think it works on windows though

Reply:
Anyway i think this works well with qbittorrent's own theme

Reply:
>Anyway i think this works well with qbittorrent's own theme

That would add another annoyance for users with dark theme.
I don't think the currently used combo box is satisfactory, but it is preferable than a custom (non-native) widget suggested here.

Reply:
But pause and other icons are of same color

Should i replace it with tree view because it's very hard to get the native icons on windows atleast

Reply:
>Should i replace it with tree view because it's very hard to get the native icons on windows atleast

Probably worth a try, there might be other unforeseen inconveniences though...

Reply:
>Should i replace it with tree view because it's very hard to get the native icons on windows atleast

It should be possible by subclassing QCheckBox and follow the steps in https://doc.qt.io/qt-5/qstyle.html#developing-style-aware-custom-widgets
And call `QStyle::drawPrimitive()` with  `PE_IndicatorBranch` and `State_Children`/`State_Open`.
Haven't tried it yet, but it looks feasible!